stress/shell/src/main/java/alluxio/stress/cli/RegisterWorkerBench.java

```diff
@@ -83,6 +83,12 @@ public String getBenchDescription() {
     ));
   }
 
+  @Override
+  public String checkIfMultipleTask()
+  {
```

Comment:
code style: No line break before the opening brace. https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style
Also in other files.